packages/utils/src/locking.js

```diff
@@ -35,16 +35,33 @@ export async function getLockPath(config) {
 export async function lock({ id, dir, root, options }) {
   const lockPath = await getLockPath({ id, dir, root })
 
-  const locked = await properlock.check(lockPath)
-  if (locked) {
-    consola.fatal(`A lock with id '${id}' already exists on ${dir}`)
+  try {
+    const locked = await properlock.check(lockPath)
+    if (locked) {
+      consola.fatal(`A lock with id '${id}' already exists on ${dir}`)
+    }
+  } catch (e) {
+    consola.warn(`Check for existing lock failed`, e)
```

Comment:
Is this warn useful for normal users? What if we use `debug` level?

Reply:
Hypothetically speaking you could argue that when you were unable to check if a lock exists, you should assume the lock exists. But as in Nuxt' case the lock is more just a helper then a full guard, you could just warn or ignore the fact. I prefer the warning, but probably more from the hypothetical standpoint then a practical one. But I am happy to change it if you prefer

Reply:
If it is a safe thing to pass probably hiding for normal users would be better DX. Anyway choose the log level that you prefer :)